src/lib/core/url.ts

```diff
@@ -10,7 +10,13 @@ import { SDKOptions } from '../services/types.js';
  * @private
  */
 export function stringifyParameters(parameters: {
-  [key: string]: string | number | Array<string | number | null | undefined> | null | undefined;
+  [key: string]:
+    | string
+    | number
+    | boolean
```

Comment:
I added `boolean` in response to a type failure with one of the endpoint-manager task filter values.